hephaistus/main-theme/distribution-system/chord/playground/nullist0/lib/src/main/kotlin/org/lia/dht/nullist0/chord/event/ChordEvent.kt

```diff
@@ -0,0 +1,13 @@
+package org.lia.dht.nullist0.chord.event
+
+import org.lia.dht.nullist0.chord.ChordConfiguration
+
+/**
+ * An interface to represent the domain event of chord algorithm.
+ */
+internal interface ChordEvent<Id, Value, ReturnType>: Function<ReturnType> {
```

Comment:
Is there a plan to create `JoinToChordRing`?

Reply:
Thank you for implementing interesting approach 😄

Reply:
Yes there is a plan to create `JoinToChordRing` and `LeaveFromChordRing`.

Reply:
Nice 🙂